test/acceptance/helpers/index.js

```diff
@@ -138,6 +139,15 @@ const startGravity = (port) =>
         expires_in: moment().add(100, 'days').utc().format()
       })
     })
+    app.get('/api/v1/profile/pace-gallery', (req, res) => {
+      res.send(fabricate('profile', {
+        owner_type: 'PartnerGallery',
+        owner: fabricate('partner', {
+          id: 'pace-gallery',
+          profile_layout: 'gallery_one'
+        })
+      }))
+    })
```

Comment:
I think we may be able to move it to antigravity since it already has [`'/api/v1/profile/:id'`](https://github.com/artsy/antigravity/blob/bcd6f6b/node/server.coffee#L70). What do you think?